core/domain/feedback_services.py

```diff
@@ -261,3 +263,40 @@ def enqueue_feedback_message_email_task(user_id):
     taskqueue_services.enqueue_task(
         feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
+
+
+def get_feedback_message_references(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=True)
+
+    return [feedback_domain.FeedbackMessageReferences(
+        reference['exploration_id'], reference['thread_id'],
+        reference['message_id'])
+            for reference in model.feedback_message_references]
+
+def update_feedback_message_reference(user_id, reference):
```

Comment:
Also do you plan to use this elsewhere? If not, maybe make it a private function and declare it within send_feedback_message_email(), like you did before. This ensures that other clients don't call it and forget to put it in a transaction.

Alternatively you can have a top-level add_feedback_message_reference function but I would then suggest moving the run_in_transaction() call into it, and having a private _add_feedback_message_reference function within this one, for similar reasons.